analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -177,7 +177,11 @@ open class Npm(
             licenseNode["type"]?.textValue()
         }
 
-        return declaredLicenses
+        return declaredLicenses.map {
+            // NPM does not mean https://unlicense.org/ here, but the wish to not "grant others the right to use a
+            // private or unpublished package under any terms", which corresponds on SPDX' "NONE".
+            if (it == "UNLICENSED") "NONE" else it
+        }.toSortedSet()
```

Comment:
Better use `mapTo(sortedSetOf())` to avoid creating a temporary collection.

Reply:
Done.